src/index.ts

```diff
@@ -0,0 +1,168 @@
+import * as _ from 'underscore';
+import { AWSProvider, ServerlessInstance, ServerlessConfig } from './types';
+
+interface PluginConfig {
+  stages?: string[];
+  roleArn?: string;
+  filterPattern: string;
+  normalizedFilterID: boolean;
+  createLambdaPermission: boolean;
+  destinationARN: string;
+}
+
+const configDefaults = {
```

Comment:
```suggestion
const CONFIG_DEFAULTS = {
```
Or you can make it as a part of the plugin class

Reply:
addressed